exporter/chronicleexporter/marshal.go

```diff
@@ -113,14 +114,54 @@ func (m *protoMarshaler) extractRawLogs(ctx context.Context, ld plog.Logs) (map[
 					Data:           []byte(rawLog),
 				}
 				entries[logType] = append(entries[logType], entry)
+				// each logType maps to exactly 1 namespace value
+				if _, ok := namespaceMap[logType]; !ok {
+					namespaceMap[logType] = namespace
+				}
```

Comment:
This should also check if the namespace is empty before adding
```suggestion
                         if namespace != "" {
				// each logType maps to exactly 1 namespace value
				if _, ok := namespaceMap[logType]; !ok {
					namespaceMap[logType] = namespace
				}
			}
```